module/data/item/race.mjs

```diff
@@ -0,0 +1,73 @@
+import SystemDataModel from "../abstract.mjs";
+import { AdvancementField, IdentifierField } from "../fields.mjs";
+import ItemDescriptionTemplate from "./templates/item-description.mjs";
+
+/**
+ * Data definition for Race items.
+ * @mixes ItemDescriptionTemplate
+ *
+ * @property {string} identifier       Identifier slug for this race.
+ * @property {object[]} advancement    Advancement objects for this race.
+ * @property {number} darkvision       Darkvision distance.
+ * @property {object} movement
+ * @property {number} movement.burrow
+ * @property {number} movement.climb
+ * @property {number} movement.fly
+ * @property {number} movement.swim
+ * @property {number} movement.walk
+ * @property {string} movement.units   Units used to measure movement.
+ * @property {object} type
+ * @property {string} type.value       Creature type (e.g. "humanoid", "construct").
+ * @property {string} type.subtype     Additional type data (e.g. "elf" for Eladrin)
+ */
+export default class RaceData extends SystemDataModel.mixin(ItemDescriptionTemplate) {
+  /** @inheritdoc */
+  static defineSchema() {
+    return this.mergeSchema(super.defineSchema(), {
+      identifier: new IdentifierField({label: "DND5E.Identifier"}),
+      advancement: new foundry.data.fields.ArrayField(new AdvancementField(), {label: "DND5E.AdvancementTitle"}),
+      darkvision: new foundry.data.fields.NumberField({min: 0, integer: true, label: "DND5E.SenseDarkvision"}),
+      movement: new foundry.data.fields.SchemaField({
+        burrow: new foundry.data.fields.NumberField({
+          initial: null, min: 0, integer: true, label: "DND5E.MovementBurrow"
+        }),
+        climb: new foundry.data.fields.NumberField({
+          initial: null, min: 0, integer: true, label: "DND5E.MovementClimb"
+        }),
+        fly: new foundry.data.fields.NumberField({
+          initial: null, min: 0, integer: true, label: "DND5E.MovementFly"
+        }),
+        swim: new foundry.data.fields.NumberField({
+          initial: null, min: 0, integer: true, label: "DND5E.MovementSwim"
+        }),
+        walk: new foundry.data.fields.NumberField({
+          initial: 30, min: 0, integer: true, label: "DND5E.MovementWalk"
+        }),
+        units: new foundry.data.fields.StringField({initial: "ft", label: "DND5E.MovementUnits"})
+      }),
+      type: new foundry.data.fields.SchemaField({
+        value: new foundry.data.fields.StringField({initial: "humanoid", label: "DND5E.CreatureType"}),
+        subtype: new foundry.data.fields.StringField({label: "DND5E.CreatureTypeSelectorSubtype"})
+      })
+    });
+  }
+
+  /* -------------------------------------------- */
+
+  prepareDerivedData() {
```

Comment:
I know we've done this elsewhere for other models, but it's probably worth considering whether we really *need* this system of prepared `labels` here. Feels like we could prepare labels elsewhere (like in the Race config sheet) without needing them to be cached on the `Item` instance.